modules/navier_stokes/examples/pipe_mixing_length/pipe_mixing_length.i

```diff
@@ -0,0 +1,260 @@
+# This example demonstrates how the mixing length model can be tuned to match an
```

Comment:
It has been my personal experience that untested input files inevitably stop working over time. What we often do for big examples is a `check_input` test that looks like this:
```
[Tests]
  [pipe_mixing]
    type = RunApp
    input = 'pipe_mixing_length.i'
    check_input = True
  []
[]
```
where we can at least be confident that all the syntax in the input file still works, making it more likely that this example will run two years from now when a user wants to try out the example. I can't recall whether `check_input` actually checks for the existence of the mesh file...it probably does. If so, and the `.msh` file we would need is huge, then perhaps we can comment out the `FileMeshGenerator` and put in a simpler stand-in?

Reply:
The `.msh` file weighs 500 kB so it's probably bigger than what we want to commit into the repo. I'd prefer that we don't comment out the real mesh; I'm worried that users would overlook it in the input file. Do you think we could modify the testing infrastructure so we could have something like `gmsh_input = mesh/pipe.geo` to have the test harness itself run gmsh?

Reply:
The latter would probably be easier and more sustainable. Otherwise we will have to make sure that gmsh is available on our testing boxes

Reply:
Actually I'm not sure...I don't know how much `check_input` does. For example does it want to do kernel and material integrity checks? If so, then we kind of need the mesh. I see that spack does actually have gmsh so maybe what you're asking for is not impossible

Reply:
Turns out that `GeneratedMeshGenerator` includes a `bias_y` parameter! With that we don't need gmsh at all for this example.